test/extended/apiserver/api_requests.go

```diff
@@ -115,7 +115,7 @@ var _ = g.Describe("[sig-arch][Late]", func() {
 				"cluster-autoscaler-operator":            44,
 				"cluster-baremetal-operator":             31,
 				"cluster-image-registry-operator":        119,
-				"cluster-monitoring-operator":            33,
+				"cluster-monitoring-operator":            124,
```

Comment:
this is a big increase.  is it well understood why the number of watches increased so much?

Reply:
4.9 threshold was computed by taking maximum from the last N jobs. 4.10 uses percentiles computed from jobs ran in the last 2 months. Still, both approaches are experimental. Unfortunately, there's currently no automation that would adjust the bounds periodically. So it is possible the 4.9 upper bound for CMO is more than enough.